specification/ws.extensions/x-kaazing-ping-pong/src/main/scripts/org/kaazing/specification/ws.extensions/x-kaazing-ping-pong/should.not.escape.binary.frame/request.rpt

```diff
@@ -0,0 +1,39 @@
+property key ${ws:handshakeKey()}
+property writeMask ${ws:randomBytes(4)}
+
+connect http://localhost:8001/echo
+connected
+
+write method "GET"
+write version "HTTP/1.1"
+write header host
+write header "Upgrade" "websocket"
+write header "Connection" "Upgrade"
+write header "Sec-WebSocket-Key" ${key}
+write header "Sec-WebSocket-Version" "13"
+write header "Sec-WebSocket-Extensions" "x-kaazing-ping-pong"
+
+read status "101" /.+/
+read version "HTTP/1.1"
+read header "Upgrade" /(?i:websocket)/
+read header "Connection" /(?i:Upgrade)/
+read header "Sec-WebSocket-Accept" ${ws:handshakeHash(key)}
+# The 4 control bytes for ping-pong extension are 0x01010102:
```

Comment:
Great, thanks. You could have just used the same PR but a new one is fine too. I'll merge the new one and close this one since the new one already contains all the changes from this one.